redwood-lazylayout-composeui/src/commonMain/kotlin/app/cash/redwood/lazylayout/composeui/ComposeUiLazyList.kt

```diff
@@ -123,7 +123,7 @@ internal class ComposeUiLazyList :
     this.pullRefreshContentColor = Color(pullRefreshContentColor.toLong())
   }
 
-  override val value = @Composable {
+  override val value: @Composable () -> Unit = @Composable {
```

Comment:
This also works around https://youtrack.jetbrains.com/issue/KT-68850, but I didn't feel the need to comment.